schedule/virt_autotest/install_guest_on_slem_kvm_host.yaml

```diff
@@ -16,7 +20,7 @@ vars:
   PATTERNS: default,kvm
   MAX_JOB_TIME: 10800
 schedule:
-  # - "{{bootup_and_install}}"
+  - "{{bootup_and_install}}"
```

Comment:
Below, `bootup` and `install` are used separately in conditional schedule. Here "{{bootup_and_install}}" is used. Do they work with each other ? Never used this way before.

Reply:
Yes. I think so, because I already do verification run with this yaml.